code/addons/docs/src/blocks/blocks/Canvas.tsx

```diff
@@ -78,6 +78,7 @@ export const Canvas: FC<CanvasProps> = (props) => {
     props.additionalActions ?? story.parameters.docs?.canvas?.additionalActions;
   const sourceState = props.sourceState ?? story.parameters.docs?.canvas?.sourceState ?? 'hidden';
   const className = props.className ?? story.parameters.docs?.canvas?.className;
+  const inline = props.story?.inline ?? story.parameters?.docs?.story?.inline ?? false;
```

Comment:
The default value of inline should be true
```suggestion
  const inline = props.story?.inline ?? story.parameters?.docs?.story?.inline ?? true;
```